collector/lib/CollectorService.h

```diff
@@ -7,11 +7,11 @@
 #include "CollectorStats.h"
 #include "Control.h"
 #include "DriverCandidates.h"
-#include "SysdigService.h"
+#include "system-inspector/Service.h"
 
 namespace collector {
 
-class SysdigService;
+class Service;
```

Comment:
A nitpick, but can this be referred to by the fully qualified name, with the namespace? Otherwise the name "Service" is quite generic, and if it could be consistently used everywhere with the "system_inspector" (as everywhere else in this PR), that would be great.